usagov/usagov-2021
# Usagov 621 remove footer blocks

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-621

## Description

Moves the footer blocks out of the cms and only into the codebase. The config changes for the cms are included. The footers for both english and spanish are now also in the same twig files, so only 3 files for the footer are needed in total. Most of the change was actually removing extra files. 

##Testing Instructions
- Make sure to import config. 
- Upon import in http://localhost/admin/structure/block/block-content all footer and identifier blocks should be gone. 
- In code twig templates are now updated in the pre-existing html-blocks directory (web/themes/custom/usagov/html-blocks). Newer duplicate twig files in templates have been removed. 
- The footer should still appear on all pages. 
- Make sure the javascript for mobile is still firing. Going to a mobile width <420px should result in the drop down for the primary footer. On desktop titles should be h3 elements. 


## Checklist for the Developer



## Checklist for the Peer Reviewers
